### PR TITLE
Improve the example

### PR DESCRIPTION
**Pull request by ggleyzer**

### CODE REVIEW

**Review by lagergren — CHANGES_REQUESTED** [review]
#Something is wrong here, while it looks like a branch from mine it still looks like code only changed by me is still marked as new. This shouldn't be the case with the plugin branch as parent. Have you transplanted this branch from one patent to another and/or cherry picked anything from anywhere? I will investigate but I have a feeling this should not go upstream until I understand where this came from.

**Comment by ggleyzer** [comment]
When I created the PR, there were no conflicts, As far as I can see, you pushed something that conflicted with my changes before merging, so naturally it became a conflict. It's quite possible that I don't really understand the proper process; let's talk about it

File: dev/build.gradle.kts.minimal
[the remark this answers is not in the document]
Comment (ggleyzer) [reply dev/build.gradle.kts.minimal:1]: The other one was am orphan "*"

File: dev/src/main/x/DevExample.x
Comment (ggleyzer) [inline dev/src/main/x/DevExample.x:1]: Changed the test code to a better looking and more modern XTC style

**Pull request closed without merging**